core/ts/MaciState.ts

```diff
@@ -1563,34 +1559,13 @@ const genSubsidyVkSig = (
     _stateTreeDepth: number,
     _intStateTreeDepth: number,
     _voteOptionTreeDepth: number,
-): BigInt => {
+): bigint => {
     return (BigInt(_stateTreeDepth) << BigInt(128)) +
            (BigInt(_intStateTreeDepth) << BigInt(64)) +
             BigInt(_voteOptionTreeDepth)
 }
 
 
-/*
- * A helper function which hashes a list of results with a salt and returns the
- * hash.
- *
- * @param results A list of vote weights
- * @parm salt A random salt
- * @return The hash of the results and the salt, with the salt last
- */
-const genTallyResultCommitment = (
```

Comment:
Why are we deleting this function?

Reply:
Ah, I see we moved it to `maci-crypto` & renamed `genTreeCommitment`.

Reply:
Correct, this function seems to better live in crypto and could also help with this: https://github.com/privacy-scaling-explorations/maci/issues/781

Reply:
Also the name was changed as imo a more general name is appropriate